Tasks/VSBuildV1/task.json

```diff
@@ -158,6 +158,25 @@
             "required": false,
             "helpMarkDown": "If true - enables default logger for msbuild",
             "groupName": "advanced"
+        },
+        {
+            "name": "isCustomVersion",
+            "type": "boolean",
+            "label": "Set custom version VS",
+            "defaultValue": "false",
+            "required": false,
+            "helpMarkDown": "Set custom version Visual Studio.",
+            "groupName": "advanced"
+        },
+        {
+            "name": "customVersion",
+            "type": "string",
+            "label": "Custom Version",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Set custom version Visual Studio.",
```

Comment:
I think it makes sense to add an example to the markdown helper for this input field